crates/sui/src/client_commands.rs

```diff
@@ -294,7 +295,20 @@ impl SuiClientCommands {
                 let sender = context.try_get_object_owner(&gas).await?;
                 let sender = sender.unwrap_or(context.active_address()?);
 
-                let compiled_modules = build_move_package_to_bytes(&package_path, build_config)?;
+                let compiled_modules = build_move_package_to_bytes(&package_path, build_config.clone())?;
```

Comment:
What I would suggest: call [`build_config.build_package(&package_path, &mut Vec::new())`](https://github.com/move-language/move/blob/main/language/tools/move-package/src/lib.rs#L151) here instead. This will give you a `CompiledPackage` value, and you can get the bytecodes for dependencies from [this](https://github.com/move-language/move/blob/main/language/tools/move-package/src/compilation/compiled_package.rs#L83) field.

Note (probably for @tnowacki /@awelc): I think we need to provide a wrapper for the `build_package` that also runs the Sui verifier on all of the output.